src/app/(nousSuivre)/blog/page.js

```
@@ -225,13 +225,13 @@ Grâce à la générosité de la Fondation Achraf Hakimi, nos jeunes ont été a
                 TelQuel parle de nous !
               </h1>
               <p className="text-gray-600 mb-4 text-sm md:text-base leading-relaxed">
-                L’association Bab Rayan a récemment été mise en lumière par
+                L'association Bab Rayan a récemment été mise en lumière par
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)